tests/framework/pod.go

```diff
@@ -0,0 +1,30 @@
+package framework
```

Comment:
So far,  I think this file should be moved to _tests/utils/_

Reply:
Its more a convenience than anything, I like it because I don't have to pass f.K8sClient and f.Namespace constantly. Of course it sort of doesn't work for the cloner since that needs 2 namespaces, but everywhere else it is convenient.

Reply:
I have trouble with these as well.  The first, like you point out, is that it's not applicable to all test cases.  Every cloner test will be all utility calls or a mix of util calls and framework utility wrappers.  The second is that it doesn't seem to buy us much at the cost of adding code that will have to be maintained and (maybe more importantly) scaled as more utility functions are added.  For instance, I see #328 (Datavol e2e) does the same.